crates/sui-core/src/event_filter.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use move_core_types::account_address::AccountAddress;
+use move_core_types::identifier::Identifier;
+use move_core_types::language_storage::StructTag;
+use serde::Deserialize;
+use serde::Serialize;
+use std::collections::BTreeMap;
+use sui_types::base_types::SuiAddress;
+use sui_types::event::{Event, EventEnvelope};
+
+#[derive(Eq, PartialEq, Ord, PartialOrd, Clone, Debug, Serialize, Deserialize)]
```

Comment:
This was added because the initial thoughts was to expose this in the RCP API, but now we decided to keep this internal and create a new `SuiEventFilter` if we decide to expose this in the API, will remove this!